roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -42,6 +42,24 @@
       ansible.builtin.debug:
         msg: "{{ ceph_orch_ls.stdout_lines }}"
       when: cifmw_cephadm_verbose | bool
+
+    - name: Get ceph config dump
+      ansible.builtin.command: "{{ cifmw_cephadm_ceph_cli }} config dump"
+      become: true
+      register: ceph_config_dump
+    - name: Show config dump
+      ansible.builtin.debug:
+        msg: "{{ ceph_config_dump.stdout_lines }}"
+      when: cifmw_cephadm_verbose | bool
+
+    - name: Get ceph require-min-compat-client
+      ansible.builtin.command: "{{ cifmw_cephadm_ceph_cli }} osd get-require-min-compat-client"
+      become: true
+      register: ceph_min_compat_client
+    - name: Show ceph require-min-compat-client
+      ansible.builtin.debug:
+        msg: "{{ ceph_min_compat_client.stdout_lines }}"
+
```

Comment:
@katarimanojk  Would you please add `ceph --version` to this output?

For example
```
[root@compute-0 ~]# ceph --version
ceph version 18.2.1-228.el9cp (26a6dfb9d5132badb606ef83e88da470a80452a6) reef (stable)
[root@compute-0 ~]# 
```

Reply:
Done